examples/autokey/autokey-setup/README.md

```diff
@@ -0,0 +1,25 @@
+# Autokey Example
+
```

Comment:
The PR's description should start with `feat!:` since this is a breaking change. (By doing that, the release bot will update the version from 3.2 to 4.0)

Reply:
thanks done